src/channel/sink_impl.rs

```diff
@@ -74,3 +75,73 @@ impl<T> Sink<T> for &UnboundedSender<T> {
         Poll::Ready(Ok(()))
     }
 }
+
+pub trait QuickSinkExt<Item>: Sink<Item> {
+    fn quick_start_send(&mut self, item: Item) -> Result<(), Self::Error>;
+
+    fn quick_send(&mut self, item: Item) -> QuickSend<'_, Self, Item>
+    where
+        Self: Unpin,
+    {
+        QuickSend::new(self, item)
+    }
+}
+
+impl<T> QuickSinkExt<T> for Sender<T> {
+    fn quick_start_send(&mut self, msg: T) -> Result<(), Self::Error> {
+        (*self).start_quick_send(msg)
```

Comment:
What's the difference between `quick_start_send` and `start_quck_send`? Can they use the same name to be consistent?